features/admin.users.v1/components/wizard/steps/add-user-basic.tsx

```diff
@@ -367,6 +582,227 @@ export const AddUserUpdated: React.FunctionComponent<AddUserProps> = (
+                                    };
+                                } else {
+                                    opValue = {
+                                        [schemaId]: {
+                                            [schemaNames[0]]: schema.type.toUpperCase() === "BOOLEAN" ?
+                                                !!values.get(schema.name)?.includes(schema.name) :
+                                                values.get(schemaNames[0])
+                                        }
+                                    };
+                                }
+                            } else {
+                                opValue = schemaNames[0] === UserManagementConstants.SCIM2_SCHEMA_DICTIONARY
+                                    .get("EMAILS")
+                                    ? { emails: [ values.get(schema.name) ] }
+                                    : schemaNames[0] === UserManagementConstants.SCIM2_SCHEMA_DICTIONARY
+                                        .get("LOCALE")
+                                        ? { [schemaNames[0]]: normalizeLocaleFormat(
+                                            values.get(schemaNames[0]) as string,
+                                            LocaleJoiningSymbol.UNDERSCORE,
+                                            false,
+                                            supportedI18nLanguages
+                                        ) }
+                                        : { [schemaNames[0]]: values.get(schemaNames[0]) };
+                            }
+                        } else {
+                            if (schema.extended && schema.multiValued) {
+                                opValue = {
+                                    [schema.schemaId]: {
+                                        [schemaNames[0]]: constructPatchOpValueForMultiValuedAttribute(
+                                            schemaNames[1],
+                                            multiValuedAttributeValues[schema.name],
+                                            multiValuedInputFieldValue[schema.name]
+                                        )
+                                    }
+                                };
+                            } else if (schema.extended) {
+                                const schemaId: string = schema?.schemaId
+                                    ? schema.schemaId
+                                    : userConfig.userProfileSchema;
+
+                                opValue = {
+                                    [schemaId]: {
+                                        [schemaNames[0]]: {
+                                            [schemaNames[1]]: schema.type.toUpperCase() === "BOOLEAN" ?
+                                                !!values.get(schema.name)?.includes(schema.name) :
+                                                values.get(schema.name)
```

Comment:
Formatting suggestion
```suggestion
                                            [schemaNames[1]]: schema.type.toUpperCase() === "BOOLEAN"
                                                ? !!values.get(schema.name)?.includes(schema.name)
                                                : values.get(schema.name)
```

Reply:
Fixed with 5cbff8089174971c91a1c1aabd4c1a11270bdb83